llvm/test/MC/ELF/debug-loc-label.s

```diff
@@ -0,0 +1,101 @@
+// Verify that the .loc_label instruction resets the line sequence and generates
+// the requested label at the correct position in the line stream
+
+// RUN: llvm-mc -filetype obj -triple x86_64-linux-elf %s -o %t.o
```

Comment:
Just `-triple x86_64` to make it clear that this is generic ELF behavior, not specific to Linux.

linux-elf is kinda incorrect triple. When the os is linux, "elf" should not be used.